Readme.md

```diff
@@ -70,6 +70,23 @@ Specifies the transaction field where gathered barcode information should be sto
 
 *Default*: `ItemEdition`
 
+### ReplaceVolumeWhenNotNull
+
+Determines if the specified volume destination field is replaced if there is already set information in the field.
+
+*Default*: `false`
+
+### ExactSearch
+
+Determines whether searching is exact (i.e., looks for an exact string match). If set to false, searching would rather rely on substring matching.
+
+*Default*: `true`
+
+### CallNumberFieldRegularExpression
+
+A regular expression for parsing the call number field for volume information. If set to nothing, full string of field is used.
```

Comment:
We should probably rename this setting to "VolumeSourceFieldRegularExpression", also, proposed rewording.

> A regular expression for parsing the volume from the field specified in the `VolumeSourceField` setting. The matched result will be used instead of the value from the `VolumeSourceField`, whenever volume information is compared. If this is set to blank, the actual value from the `VolumeSourceField` will be used.